app/helpers/audits_helper.rb

```diff
@@ -77,7 +77,7 @@ def details(audit, path = audit_path(audit))
           to = HostStatus::Global.new(base[1]).to_label
           _("Global status changed from %{from} to %{to}") % { :from => from, :to => to }
         else
-          _("%{name} changed from %{label1} to %{label2}") % { :name => name.humanize, :label1 => id_to_label(name, change[0]), :label2 => id_to_label(name, change[1]) }
+          _("%{name} changed from %{label1} to %{label2}") % { :name => name.humanize, :label1 => id_to_label(name, change[0], :audit => audit, :truncate => true), :label2 => id_to_label(name, change[1], :audit => audit, :truncate => true) }
```

Comment:
no need to set `:truncate => true` since that's the default value

Reply:
The default value is ` { :audit => @audit, :truncate => true }` - if I don't set `:truncate => true` here, I'm overriding the default value to be `{ :audit => audit }`